salt/states/virt.py

```diff
@@ -414,42 +604,75 @@ def network_define(name, bridge, forward, **kwargs):
             - vport: openvswitch
             - tag: 180
             - autostart: True
-            - start: True
 
     '''
     ret = {'name': name,
            'changes': {},
-           'result': False,
+           'result': True,
            'comment': ''
            }
 
-    kwargs = salt.utils.args.clean_kwargs(**kwargs)
-    vport = kwargs.pop('vport', None)
-    tag = kwargs.pop('tag', None)
-    autostart = kwargs.pop('autostart', True)
-    start = kwargs.pop('start', True)
```

Comment:
Does `start` need to be deprecated? I don't see this option used below and people might be passing this through still.

Reply:
That state hasn't been published too... that's why I didn't deprecate it.

Reply:
@rallytime This one and the other one you've spot are two states that haven't been published yet... that's why start hasn't been deprecated and removed directly. And after all, what's the point in telling we want the network / pool to be started in the ``*_running`` state?